lib/src/asymmetric_operator.dart

```diff
@@ -99,12 +99,13 @@ class _AsymmetricVerifier extends Verifier<PublicKey>
       : super._(algorithm, key);
 
   @override
-  pc.Signer get _algorithm => super._algorithm;
+  pc.Signer get _algorithm => super._algorithm as pc.Signer;
 
   @override
   bool verify(Uint8List data, Signature signature) {
     if (key is RsaKey) {
-      _algorithm.init(false, pc.ParametersWithRandom(keyParameter, null));
```

Comment:
@jpeiffer @rbellens curious if you know the right way to replace this null now that we can't pass null to PC.